Lesson 6 - Developer tools/homework/package.json

```diff
@@ -0,0 +1,25 @@
+{
+
+  "name": "homework",
+  "version": "1.0.0",
+  "description": "Homework for Lesson 6",
+  "main": "test.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
It's nice thing to add `gulp` call to `npm scripts`. See more at http://blog.keithcirkel.co.uk/how-to-use-npm-as-a-build-tool/